lsp-mode.el

```diff
@@ -2555,8 +2581,8 @@ PARSER is the workspace parser used for handling the message."
       (signal 'lsp-unknown-message-type (list json-data)))))
 
 (defconst lsp--default-notification-handlers
-  (lsp-ht ("window/showMessage" 'lsp--window-show-message)
-          ("window/logMessage" 'lsp--window-show-message)
+  (lsp-ht ("window/logMessage" 'lsp--window-log-message)
```

Comment:
changing showMessage to logMessage seems to be incorrect or not relevant to the PR

Reply:
Well, I thought it would be confusing to have a function called `lsp--window-show-message` not actually show any message.

At first I'd implemented `lsp--window-log-message` alongside `lsp--window-show-message` and had `window/logMessage` and `window/showMessage` route to the appropriate, but then all the `Parsing ..` lines of php-language-server ended up in the message buffer again, so apparently it uses showMessage for those.

Reply:
Are we talking about the same thing? I mean that "window/showMessage" should not be changed to "window/logMessage".